Src/BootCamp.Chapter/Lesson4.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    class Lesson4
+    {
+        public static void Demo()
+        {
+            GetInfoAndPrint();
+            GetInfoAndPrint();
+        }
+
+        static void GetInfoAndPrint()
+        {
+            Console.Write("Name: ");
+            string name = Console.ReadLine();
+
+            var age = PromptForInt("Age: ");
+
```

Comment:
No need to have a blank line after related pieces of code. I would separate calculation and printing, but the rest seems related.